paper/mel_report.Rmd

```diff
@@ -31,37 +34,237 @@ knitr::opts_chunk$set(fig.align = 'center',
-|-----------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|----------------|
-| Outcome 4: Health providers who have participated in the ‘Responding to GBV Learning Lab’ Curriculum demonstrate improved knowledge related to GBV | Indicator 4: % of health facilities demonstrate improvement in staff knowledge, attitude, and empathy in responding to GBV | Three time points: pre-training, post-intensive training, and post-intervention                                             | Average composite score of knowledge, attitude, and empathy domains by health facility cohort pre- to post-intervention | 80% of health facilities |
-| Outcome 5: Health providers who have participated in the ‘Responding to GBV Learning Lab’ have increased confidence in delivering survivor-centered care   | Indicator 5: % of health facilities that demonstrate improvement in health provider confidence in responding to GBV | Three time points: pre-training, post-intensive training, and post-intervention                                             | Average composite score of the confidence, system support, and professional role domains  by health facility cohort pre to post-intervention | 80% of health facilities |
-| Outcome 7: Increase in clients identified who are victims of GBV                                                     | Indicator 7: % increase in GBV clients identified                                         | Two timepoints: pre-training and post-intervention* *This measure can be used to compare to health facility reporting (which is the official MEL data source) on clients identified | Percentage compared to baseline | 50% increase   |
+## Outcome 4: Health providers who have participated in the ‘Responding to GBV Learning Lab’ Curriculum demonstrate improved knowledge related to GBV.
+When assessing indicator 4, the percentage of health facilities demonstrating improvement in staff knowledge, attitude, and empathy in responding to GBV, the project exceeded the MEL target, with 90% of   facilities demonstrating improvement in staff knowledge, attitudes, and empathy in responding to GBV (table 3). Additional analysis of this indicator can be found in the Appendix.
 
```

Comment:
Edit: 
When evaluating Outcome 4, which measures the percentage of healthcare facilities showing enhancement in staff knowledge, attitudes, and empathy regarding GBV response, the project surpassed the MEL (Monitoring, Evaluation, and Learning) target. Specifically, 90% of the facilities exhibited improvements in staff knowledge, attitudes, and empathy in dealing with GBV, as detailed in Table 3. Further analysis of this indicator is available in the Appendix.